Jellyfin.Api/Helpers/ClassMigrationHelper.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Reflection;
+
+namespace Jellyfin.Api.Helpers
+{
+    /// <summary>
+    /// A static class for copying matching properties from one object to another.
+    /// TODO: remove at the point when a fixed migration path has been decided upon.
+    /// </summary>
+    public static class ClassMigrationHelper
```

Comment:
I suggest moving this to Jellyfin.Server, potentially under the `Migrations` directory

Reply:
Moved

Reply:
Just tried. Emby.Server.Implementations can't see it and can't add as circular reference.